modules/ocf_printhost/manifests/cups.pp

```diff
@@ -59,9 +59,16 @@
     }
   }
 
+  #Tea4cups saves files based on its owner
+  file { '/usr/lib/cups/backend/tea4cups':
+    ensure => 'file',
+    owner  => 'ocfenforcer',
+    mode   => '0700';
```

Comment:
Probably also good to include `require => Package['cups-tea4cups']` here since this is normally provided by that package so you'd want it to exist first before modifying it's permissions